core/src/main/java/com/novoda/merlin/Endpoint.java

```diff
@@ -0,0 +1,51 @@
+package com.novoda.merlin;
+
+public class Endpoint {
+
+    private static final Endpoint DEFAULT_ENDPOINT = Endpoint.from("http://connectivitycheck.android.com/generate_204");
+
+    private final String endpoint;
+
+    public static Endpoint defaultEndpoint() {
+        return DEFAULT_ENDPOINT;
+    }
+
+    public static Endpoint from(String endpoint) {
```

Comment:
Ah I guess I did this because I exposed default endpoint in the same way. I didn't want the client to deal with `new Endpoint(String endpoint)` and `Endpoint.defaultEndpoint()` 😬